packages/manager-react-components/package.json

```diff
@@ -117,9 +98,12 @@
     "vitest": "^1.2.0",
     "zustand": "^4.5.5"
   },
-  "optionalDependencies": {
+  "peerDependencies": {
+    "@ovh-ux/manager-core-api": "^0.9.0-alpha.0",
+    "@ovh-ux/manager-react-shell-client": "1.0.0",
     "i18next": "^23.8.2",
     "react": "^18.2.0",
+    "react-use": "^17.5.0",
```

Comment:
**nitpick:** This will cause installation errors as not all the apps are using `react-use`.

Reply:
👍